.github/workflows/ci.yaml

```diff
@@ -32,6 +32,11 @@ jobs:
             - name: Build
               run: npm run build
 
+            - name: Add missing hostname to /etc/hosts
```

Comment:
Great, can we reference the issue #1138?

Reply:
@relu91 I assume you are okay with the latest changes?